sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUser.md

```diff
@@ -40,13 +40,13 @@ Get-SPOUser [-LoginName <String>] -Site <SpoSitePipeBind> [<CommonParameters>]
 
 This cmdlet contains more than one parameter set.
 You may only use parameters from one parameter set and you may not combine parameters from different parameter sets.
-For more information about how to use parameter sets, see Cmdlet Parameter Sets (<https://go.microsoft.com/fwlink/?LinkID=187810).>
+For more information about how to use parameter sets, see [Cmdlet Parameter Sets](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/cmdlet-parameter-sets).
```

Comment:
Remove `en-us` from the URL

Reply:
Thanks for your review, I updated the PR.